features/git-hack/hack_push_strategy_local.feature

```diff
@@ -0,0 +1,31 @@
+Feature: git hack: don't push branch to remote upon creation
+
+  As a developer starting work on a new private feature
+  I don't want git-hack to push my branch to the remote repo
+  So that the upstream repo is not aware of my work
```

Comment:
It's one possible motivation, but yeah I should use the scenario that actually prompted me to do this.